sound/soc/sof/ipc4-topology.c

```diff
@@ -1706,6 +1709,11 @@ static int sof_ipc4_control_load_bytes(struct snd_sof_dev *sdev, struct snd_sof_
 		}
 	}
 
+	msg = &control_data->msg;
+	msg->primary = SOF_IPC4_MSG_TYPE_SET(SOF_IPC4_MOD_LARGE_CONFIG_SET);
```

Comment:
this is modifed in sof_ipc4_set_get_data() to SET/GET depending on whether it is a put/get. So maybe we dont need to se this here?